plaid/src/executor/mod.rs

```diff
@@ -369,7 +375,12 @@ fn execution_loop(
             let error = match entrypoint.call(&mut store) {
                 Ok(n) => {
                     if n != 0 {
-                        Some(ModuleExecutionError::ModuleErrorCode(n))
+                        Some(ModuleExecutionError::ModuleError(
+                            env.as_ref(&store)
+                                .execution_error_context
+                                .clone()
+                                .unwrap_or("> empty <".to_string()),
```

Comment:
I'd just use None instead of `> empty <` unless you have a further reason to depart from the standard `Option` verbiage.